documentation/source/theory/rmg/enlarger.rst

```diff
@@ -70,8 +70,36 @@ For a bimolecular reaction occuring between species A and B, this threshold is s
 
 :math:`R_{bimolecular} = k_{threshold}C_A C_B > \epsilon R_{char}` 
 
+where :math:`k_{threshold} = filterThreshold`. ``filterThreshold`` is set by the user in the input file and its
+default value is :math:`10^{8}`. This is on the same order of magnitude as the collision limit for two hydrogen atoms
```

Comment:
You should have units on this since they are relevant

Reply:
Good point